src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripItemTests.cs

```diff
@@ -15398,6 +15398,53 @@ public void ToolStripItem_IDropTargetOnDragOver_Invoke_CallsDragOver(DragEventAr
         Assert.Equal(1, callCount);
     }
 
+    [WinFormsFact]
+    public void ToolStripItem_OnItemSelected()
+    {
+        using var menuStrip1 = new MyMenuStrip();
+        using var toolStripMenuItem1 = new ToolStripMenuItem();
+        using var toolStripMenuItem2 = new ToolStripMenuItem();
+        using var toolStripMenuItem3 = new ToolStripMenuItem();
```

Comment:
Please use target type new:
```suggestion
        using MyMenuStrip menuStrip1 = new();
        using ToolStripMenuItem toolStripMenuItem1 = new();
        using ToolStripMenuItem toolStripMenuItem2 = new();
        using ToolStripMenuItem toolStripMenuItem3 = new();
```